Error in _prepareSkillsTools when skills deleted from CONFIG

If skills are deleted from `CONFIG.DND5E.skills`, the following error will occur on trying to open an actor sheet when said actor has an item or active effect that adds to `actor.system.skills`. For example, if the actor is wearing Chainmail with the Stealth Disadvantage property or an actor has an Active Effect adding a check bonus to a skill.

```
Uncaught (in promise) Error: Failed to render Application "CharacterActorSheet-Actor-JXCLjd3niYecVhwy":
Cannot read properties of undefined (reading 'localeCompare')
[No packages detected]
    at #render (foundry.mjs:27248:13)
    at async #try (foundry.mjs:7169:17)Caused by: TypeError: Cannot read properties of undefined (reading 'localeCompare')
    at base-actor-sheet.mjs:542:32
    at Array.sort (<anonymous>)
    at CharacterActorSheet._prepareSkillsTools (base-actor-sheet.mjs:542:9)
    at CharacterActorSheet._prepareDetailsContext (character-sheet.mjs:362:27)
    at CharacterActorSheet._preparePartContext (character-sheet.mjs:203:35)
    at async CharacterActorSheet._renderHTML (foundry.mjs:28611:29)
    at async #render (foundry.mjs:27239:22)
    at async #try (foundry.mjs:7169:17)
```

--- a/module/applications/actor/api/base-actor-sheet.mjs
+++ b/module/applications/actor/api/base-actor-sheet.mjs
@@ -515,15 +515,17 @@ export default class BaseActorSheet extends PrimarySheetMixin(
       if ( property === "skills" ) src = CONFIG.DND5E.skills[key]?.ability;
       return src ?? "int";
     };
-    return Object.entries(context.system[property] ?? {}).map(([key, entry]) => ({
-      ...entry, key,
-      abbreviation: CONFIG.DND5E.abilities[entry.ability]?.abbreviation,
-      baseAbility: baseAbility(key),
-      hover: CONFIG.DND5E.proficiencyLevels[entry.value],
-      label: (property === "skills") ? CONFIG.DND5E.skills[key]?.label : Trait.keyLabel(key, { trait: "tool" }),
-      link: { action: "roll", key, type: property === "skills" ? "skill" : "tool" },
-      source: context.source[property]?.[key]
-    })).sort((a, b) => a.label.localeCompare(b.label, game.i18n.lang));
+    return Object.entries(context.system[property] ?? {})
+      .filter(([key]) => key in CONFIG.DND5E[property])
+      .map(([key, entry]) => ({
+        ...entry, key,
+        abbreviation: CONFIG.DND5E.abilities[entry.ability]?.abbreviation,
+        baseAbility: baseAbility(key),
+        hover: CONFIG.DND5E.proficiencyLevels[entry.value],
+        label: (property === "skills") ? CONFIG.DND5E.skills[key]?.label : Trait.keyLabel(key, { trait: "tool" }),
+        link: { action: "roll", key, type: property === "skills" ? "skill" : "tool" },
+        source: context.source[property]?.[key]
+      })).sort((a, b) => a.label.localeCompare(b.label, game.i18n.lang));
   }
 
   /* -------------------------------------------- */

